buildSrc/src/main/java/org/elasticsearch/gradle/testfixtures/TestFixturesPlugin.java

```diff
@@ -128,6 +135,16 @@ public void apply(Project project) {
 
     }
 
+    private void conditionTaskByType(TaskContainer tasks, TestFixtureExtension extension, Class<? extends DefaultTask> taskClass) {
```

Comment:
Can we give this a bit more of a descriptive name? Especially all these nested lambdas are a bit hard to follow.

Perhaps `skipTasksWithDisabledFixtures`.